build.ps1

```diff
@@ -38,7 +42,23 @@ foreach ($s in $script) {
             }
 
             if (-not $noBuild) {
```

Comment:
Double negations always throws me off, would it be simple instead:

```
if ($Build) {
....
}
```

Reply:
Keeping nobuild because it is familiar to me from dotnet test